### PR TITLE
add WeatherSavingService class and repo

### LINKED ISSUE #79: Add service for saving history weather

### PR DESCRIPTION
**Pull request by SergiLomtadze**
closes #79

### CODE REVIEW

**Review by kdavletov-exadel — APPROVED** [review]

**Review by kdavletov-exadel — CHANGES_REQUESTED** [review]
changes needed

File: src/ExadelMentorship.BusinessLogic/Interfaces/Weather/IWeatherSavingService.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ExadelMentorship.BusinessLogic.Interfaces.Weather
+{
+    public interface IWeatherSavingService
+    {
+        Task SaveInDb(string cityName);
```
[the remark this answers is not in the document]
Comment (SergiLomtadze) [reply src/ExadelMentorship.BusinessLogic/Interfaces/Weather/IWeatherSavingService.cs:10]: this service will be used later in Job

[the remark this answers is not in the document]
Comment (SergiLomtadze) [reply src/ExadelMentorship.BusinessLogic/Interfaces/Weather/IWeatherSavingService.cs]: done

File: src/ExadelMentorship.Persistence/WeatherHistorySavingRepository.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+using ExadelMentorship.Persistence.Context;
+
+namespace ExadelMentorship.Persistence
+{
+    public class WeatherHistorySavingRepository : IWeatherHistorySavingRepository
+    {
+        private readonly ApplicationDbContext _dbContext;
+        private readonly IWeatherApiService _weatherApiService;
+
+        public WeatherHistorySavingRepository(ApplicationDbContext context, IWeatherApiService weatherApiService)
```
Comment (khamza85) [inline src/ExadelMentorship.Persistence/WeatherHistorySavingRepository.cs:13]: It is better if repository would not depend on anything than DbContext and some other services like logger.

**Review by khamza85 — CHANGES_REQUESTED** [review]

File: src/ExadelMentorship.Persistence/WeatherHistorySavingRepository.cs (hunk 2 above)
Comment (SergiLomtadze) [reply src/ExadelMentorship.Persistence/WeatherHistorySavingRepository.cs]: done

**Review by kdavletov-exadel — APPROVED** [review]

**Pull request merged**